modules/src/main/assembly/dist.xml

```diff
@@ -13,119 +13,33 @@
         </file>
         <file>
-             <source>${module.xml.aliases.basedir}/ogm/couchdb/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/couchdb/main</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.basedir}/ogm/cassandra/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/cassandra/${hibernate.ogm.cassandra.module.slot}</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.aliases.basedir}/ogm/cassandra/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/cassandra/main</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.basedir}/ogm/cassandra-driver/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/cassandra/driver/${hibernate.ogm.cassandra.module.slot}</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.basedir}/ogm/redis/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/redis/${hibernate.ogm.redis.module.slot}</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.aliases.basedir}/ogm/redis/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/redis/main</outputDirectory>
-             <filtered>true</filtered>
-        </file>
-        <file>
-             <source>${module.xml.basedir}/ogm/redis-driver/module.xml</source>
-             <outputDirectory>/org/hibernate/ogm/redis/driver/${hibernate.ogm.redis.module.slot}</outputDirectory>
+             <source>${module.xml.aliases.basedir}/ogm/ignite/module.xml</source>
+             <outputDirectory>/org/hibernate/ogm/ignite/main</outputDirectory>
              <filtered>true</filtered>
         </file>
     </files>
 
     <dependencySets>
-        <!-- OGM -->
         <dependencySet>
-             <useProjectArtifact>false</useProjectArtifact>
-             <outputDirectory>org/hibernate/ogm/ehcache/${hibernate.ogm.ehcache.module.slot}</outputDirectory>
-             <useTransitiveFiltering>false</useTransitiveFiltering>
-             <unpack>false</unpack>
-             <includes>
-                 <include>org.hibernate.ogm:hibernate-ogm-ehcache</include>
-                 <include>net.sf.ehcache:ehcache-core</include>
-              </includes>
+            <useProjectArtifact>false</useProjectArtifact>
+            <outputDirectory>org/hibernate/ogm/ignite/${hibernate.ogm.ignite.module.slot}</outputDirectory>
+            <useTransitiveFiltering>false</useTransitiveFiltering>
+            <unpack>false</unpack>            
+            <includes>
+                <include>org.hibernate.ogm:hibernate-ogm-ignite</include>
+                <include>org.apache.ignite:*</include>
```

Comment:
I know that this is nicer, but In past we had issues with this including library in the module without us noticing. Could you please remove the star and list each library included?

Reply:
Ok. I will try extract Apache Ignite to separate module